settings/base.py

```diff
@@ -101,7 +101,7 @@
     'django.contrib.auth.middleware.AuthenticationMiddleware',
     'django.contrib.auth.middleware.SessionAuthenticationMiddleware',
     'django.contrib.messages.middleware.MessageMiddleware',
-    'django.middleware.clickjacking.XFrameOptionsMiddleware',
+    #'django.middleware.clickjacking.XFrameOptionsMiddleware',
```

Comment:
Using this middleware will prevent loading register site into an iframe which will prevent clickjacking. And from my perspective, clickjacking is not a concern in our site.

This is the only security term that we lose from removing this middleware. Please let me know if clickjacking is something that we need to be concerned about.